WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -226,4 +226,55 @@ d_melchisedechianism_holy_order = {
 	monthly_income = 2
 
 	mercenary_type = zealots_composition
-}	
\ No newline at end of file
+}
+
+c_priscillianism_holy_order = {
```

Comment:
The Vanilla Papacy is a kingdom, and, after the Great Schism, the WTWSMS Papacy becomes a kingdom.